src/main/java/Level1.java

```diff
@@ -0,0 +1,34 @@
+
+import java.util.Scanner;
+import java.util.*;
```

Comment:
I feel importing * is a bad practice as we are uncertain to which file is  being imported. To add on, Scanner has already been added prior to importing *. Thus, this line is not needed